--- /Users/lars/Downloads/CMakeLists.txt	2023-12-21 15:49:55
+++ CMakeLists.txt	2023-12-21 16:00:19
@@ -105,7 +105,7 @@
   else()
     # Otherwise we pick all the files in the subdirectory
     # and create a scratch for them automatically
-    file(GLOB scratch_sources CONFIGURE_DEPENDS ${subdir}/[^.]*.cc)
+    file(GLOB scratch_sources CONFIGURE_DEPENDS ${subdir}/[^.]*.cc helper/[^.]*.cc)
     create_scratch("${scratch_sources}")
   endif()
 endforeach()
