--- macros-and-definitions.cmake.orig
+++ macros-and-definitions.cmake
@@ -1611,7 +1611,7 @@
   # to doubled underlines and will cause the ns3 script to fail
   string(REPLACE "//" "/" output_directory "${output_directory}")

-  set(ns3-exec-outputname ns${NS3_VER}-${target_name}${build_profile_suffix})
+  set(ns3-exec-outputname ${target_name}${build_profile_suffix})
   set(ns3-execs "${output_directory}${ns3-exec-outputname};${ns3-execs}"
       CACHE INTERNAL "list of c++ executables"
   )