--- wscript.orig
+++ wscript
@@ -758,12 +758,13 @@
 
     try:
         for filename in os.listdir("scratch"):
-            if filename.startswith('.') or filename == 'CVS':
+            if filename.startswith('.') or filename == 'CVS' or filename == 'helper':
                 continue
             if os.path.isdir(os.path.join("scratch", filename)):
                 obj = bld.create_ns3_program(filename, all_modules)
                 obj.path = obj.path.find_dir('scratch').find_dir(filename)
                 obj.source = obj.path.ant_glob('*.cc')
+                obj.source.append('../helper/quic-network-simulator-helper.cc')
                 obj.target = filename
                 obj.name = obj.target
                 obj.install_path = None
